### PR TITLE
feat: add a table to store historical trend information

### PR DESCRIPTION
**Pull request by TRebirthC**

### CODE REVIEW

File: packages/data-model/models/TrendHistory.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '../util/database.js';
+
+export default sequelize.define(
+  'GithubProjectsHistory',
```
Comment (Jiannan-dev) [inline packages/data-model/models/TrendHistory.js]: 改下名字

File: sql/trend_history.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+-- create table trend_history
+CREATE TABLE trend_history (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    project_id INT NOT NULL,
+    data_type INT NOT NULL DEFAULT 0,
```
Comment (Jiannan-dev) [inline sql/trend_history.sql]: 补充注释

**Review by Jiannan-dev — CHANGES_REQUESTED** [review]

File: packages/data-model/models/TrendHistory.js (hunk 1 above)
Comment (TRebirthC) [reply packages/data-model/models/TrendHistory.js]: 已在 [commit] [improvements: add comment for sql file and fix the name of TrendHistory](https://github.com/originjs/oss-evaluation-service/pull/675/commits/6572357c227553e9e004db412ec8d1d9b69d0d99) 中进行修改

File: sql/trend_history.sql (hunk 2 above)
Comment (TRebirthC) [reply sql/trend_history.sql]: 已在 [commit] [improvements: add comment for sql file and fix the name of TrendHistory](https://github.com/originjs/oss-evaluation-service/pull/675/commits/6572357c227553e9e004db412ec8d1d9b69d0d99) 中进行修改

**Review by Jiannan-dev — APPROVED** [review]

**Pull request merged**